metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -285,6 +285,18 @@ public void testHappyPath() {
   }
 
   @Test
```

Comment:
So, I was going to make a test passing in NULL, but the Immutable map will not take nulls, can you clarify how you would test this?